WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -253,6 +255,40 @@ public void onResume() {
     }
 
     @Override
+    public void onStart() {
+        super.onStart();
+        EventBus.getDefault().register(this);
+    }
+
+    @Override
+    public void onStop() {
+        super.onStop();
+        EventBus.getDefault().unregister(this);
+    }
+
+    @SuppressWarnings("unused")
+    public void onEvent(ReaderEvents.FollowedTagsChanged event) {
```

Comment:
I had misread the docs, incorrectly thinking that `onEvent()` always ran on the main thread. Rather than use `onEventMainThread` how about we ensure that the events are always posted on the main thread? I find that approach cleaner since we don't have to worry about which thread the event was posted from.